src/main/kotlin/com/terraformation/backend/accelerator/db/VoteStore.kt

```diff
@@ -0,0 +1,81 @@
+package com.terraformation.backend.accelerator.db
+
+import com.terraformation.backend.accelerator.model.VoteModel
+import com.terraformation.backend.auth.currentUser
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.VoteOption
+import com.terraformation.backend.db.accelerator.tables.daos.ProjectVotesDao
+import com.terraformation.backend.db.accelerator.tables.references.PROJECT_VOTES
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import jakarta.inject.Named
+import java.time.InstantSource
+import org.jooq.DSLContext
+
+@Named
+class VoteStore(
+    private val clock: InstantSource,
+    private val dslContext: DSLContext,
+    private val projectVotesDao: ProjectVotesDao,
+) {
+  fun fetchAllVotesByProject(projectId: ProjectId): List<VoteModel> {
+    requirePermissions { updateProjectVotes(projectId) }
```

Comment:
I believe users with the "Read Only" global role are supposed to be able to view votes, so this might need a separate `readProjectVotes` permission.